suites/quincy/rados/tier-2_rados_test-bugfixes.yaml

```diff
@@ -204,3 +205,9 @@ tests:
   #     module: test_mon_connection_scores.py
   #     polarion-id: CEPH-83602911
   #     comments: Active bug 2151501
+
+  - test:
+      name: Verification of configuration values via cephadm-ansible module
```

Comment:
please shorten the name of test case.

Reply:
Updated test name 
https://github.com/red-hat-storage/cephci/blob/262205193a7497c6e25014f51a479518e7a1ba85/suites/quincy/rados/tier-2_rados_test-bugfixes.yaml#L209-L213